content/_releases/22/16.md

```diff
@@ -43,8 +43,8 @@ tenant configuration flag called `forced_totp` to switch this feature on.
 *   If you remove the `enabled_totp` feature flag, system will remove `forced_totp` as well.
 
 You can set these parameters either during the [tenant creation]({{site.data.tenant.apiDocsUri}}/v2/#create-a-tenant) or
-[update the parameters]({{site.data.tenant.apiDocsUri}}/v2/#update-a-tenant) of existing tenant via an API call following this example
```

Comment:
or you can update the parameters of an existing tenant via an APi call. An example of such an API call is shown below.

Reply:
corrected and added into a separate pull request